ibm/service/power/resource_ibm_pi_volume_test.go

```diff
@@ -160,3 +160,64 @@ func testAccCheckIBMPIVolumePoolConfig(name string) string {
 	  }
 	`, name, acc.Pi_cloud_instance_id)
 }
+
+func TestAccIBMPIVolumeGRS(t *testing.T) {
```

Comment:
What is GRS?
A func comment would like understand the test functionality.

Reply:
volume replication comes under global replication service(GRS)